src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -68,7 +68,7 @@
     private final Map<String, VCFFormatHeaderLine> mFormatMetaData = new LinkedHashMap<String, VCFFormatHeaderLine>();
     private final Map<String, VCFFilterHeaderLine> mFilterMetaData = new LinkedHashMap<String, VCFFilterHeaderLine>();
     private final Map<String, VCFHeaderLine> mOtherMetaData = new LinkedHashMap<String, VCFHeaderLine>();
-    private final List<VCFContigHeaderLine> contigMetaData = new ArrayList<VCFContigHeaderLine>();
+    private final Map<String, VCFContigHeaderLine> contigMetaData = new LinkedHashMap<>();
```

Comment:
We should probably the type here to a LinkedHashMap since we definitely do care about the ordering.  Honestly, these should all probably be declared as LinkedHashMaps. It's gross that there isn't an interface type for this sort of sorted Map, but I think it's better to put the type as what we actually require than pretending we can substitute any map.